README.md

```diff
@@ -67,36 +67,17 @@ Currently we are manually building an archive in xCode to distribute the app.
 4. Select Product > Archive to create the app archive.
 5. If your archive builds correctly it will walk you through pushing this to the app store from there. 
 
-### Running on device with local web
-1. Run web on your local machine as usual
-
-2. install ngrok
-```
-brew install ngrok
-```
-1. Create a free account here https://ngrok.com/ and obtain your auth token
-4. Configure ngrok with your token
-```
-ngrok config add-authtoken YOUR_TOKEN
-```
-3. run ngrok, note the public `Forwarding` domain listed
-```
-ngrok http 3000 # substitute port if necessary
-```
-4. Modify `.env` to point the WebView at the ngrok endpoint created above
-```
-SHAPESHIFT_URI=https://523d-135-134-162-133.ngrok.io # example ngrok domain
-```
-5. Run the app on your device from Xcode _or_ use the yarn script:
+### Running on Expo Go with local web
+- Run web, select the lan IP
+- Update your `EXPO_PUBLIC_LOCAL_URI` in the .env with the related IP
+- Run the mobile app using this command (note the --reset-cache option, it's necessary for Metro to reload the .env file):
 ```
-yarn ios:device
+npx expo start --reset-cache
 ```
-for now, the above runs `node_modules/.bin/react-native run-ios --device 'iPhone'` - substitute the name of your device for iPhone
-#### __Notes__
-- you may need to kill and reload the app after it's been completely loaded to get it to source content from metro
-- you may need to remove the line `messageManager.webviewRef?.reload()` in the useKeepAliveHook if the device has trouble downloading bundle.js before getting whacked.
-- you will need to manually add your ngrok domain to the `navigationFilter` to prevent opening in a browser. dev flag/mode?
-- as of 10/6: in your local web, move the entire contents of `react-app-rewired/headers/csps` out of their normal place and full restart web (yarn dev)
+- Install Expo Go
```

Comment:
Probably worth adding Expo go as a prerequisite too, I do have it installed on iOS and know what it is but may not be obvious

Reply:
I completely updated the readme in [4c0cdc4](https://github.com/shapeshift/mobile-app/pull/109/commits/4c0cdc4bcd8120fe53ff1c6b5e319247d80e3d1d)

Reply:
We're now missing Android section, is this expected?

Also probably worth changing the heading of `iOS` explaining what`iOS` (and TBD Android sections) is about - reading this, I would think that I need to refer to `IOS` in addition to `Launching the app` i.e run `yarn ios` if I want to test iOS bits, but reading expo docs, this seems to only be needed for a build and run, we should really only be the case in some very specific cases, is that correct understanding

Reply:
@NeOMakinG still pendarooing

Reply:
You are right, it's needed to launch the build on an simulator, I moved all the ios bits and added the android command in this commit

[13d149d](https://github.com/shapeshift/mobile-app/pull/109/commits/13d149d74f87d32090c24b82a83f6c9622a42f0e)

Most of the time we should be using exop go to develop now that we are compatible with it